src/servers/src/error.rs

```diff
@@ -214,6 +214,16 @@ pub enum Error {
         error: snap::Error,
     },
 
+    #[snafu(display("Failed to send prometheus remote request"))]
+    SendPromRemoteRequest {
+        location: Location,
+        #[snafu(source)]
+        error: reqwest::Error,
+    },
+
+    #[snafu(display("Invalid prometheus remote write config, msg: {}", msg))]
+    InvalidRemoteWriteConfig { msg: String, location: Location },
```

Comment:
Change this "remote write" as well. We have a remote write service, this naming is very confusing